sklearn/datasets/species_distributions.py

```diff
@@ -72,9 +72,8 @@ def _load_coverage(F, header_length=6, dtype=np.int16):
     header = dict([make_tuple(line) for line in header])
 
     M = np.loadtxt(F, dtype=dtype)
-    nodata = header[b'NODATA_value']
+    nodata = int(header[b'NODATA_value'])
```

Comment:
It was to fix a numpy warning that does not like indexing by float values. The value is `128.0`. I do not really understand this code but that should not impact the behavior besides the warning.